client/src/components/MainPage/TranslationsPage/index.jsx

```diff
@@ -30,6 +30,27 @@ class TranslationsPage extends Component {
     });
   }
 
+  voteUpClick = (translationsId) => {
+    const { match } = this.props;
+    const { params } = match;
+    const { questionId } = params;
+    const voteUp = 1;
+    const voteDown = 0;
+    const dataVote = { voteUp, voteDown, translationsId };
+    axios
+      .post(`/api/v1/questions/${questionId}/voteUp`, dataVote)
```

Comment:
endpoint should be using the ```lisp-case``` naming convention for better SEO.
data object should be named ```voteData```
```suggestion
      .post(`/api/v1/questions/${questionId}/vote-up`, voteData)
```